views/layouts/main.handlebars

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="css/styles.css">
+	<script src="https://kit.fontawesome.com/d2dbc0198b.js" crossorigin="anonymous"></script>
```

Comment:
Lets just use individual SVGs instead of relying on a full icon library. We'll only have like 5 icons anyway